pkg/controller/releasemanager/syncer.go

```diff
@@ -348,16 +348,6 @@ func (r *ResourceSyncer) syncSLORules(ctx context.Context) error {
 // currentTrafficPolicy gets the latest release's traffic policy, or if there
 // are no releases, then the latest revisions traffic policy.
 func (r *ResourceSyncer) currentTrafficPolicy() *istiov1alpha3.TrafficPolicy {
```

Comment:
can this whole function get removed entirely?

Reply:
yeah